deploy/base/tunedmodel.yaml

```diff
@@ -0,0 +1 @@
+tuned_model: ft_final_v20230624010424.ft_intermediate_v20230624010339.all-MiniLM-L6-v2
```

Comment:
Call this file tuned_models.yaml?

Reply:
nit: or call it models.yaml?  the key has tuned_model.  maybe in future we could add more.

Reply:
renamed to `models.yaml`